R/miic.R

```diff
@@ -92,17 +92,21 @@
 #' propagated to downstream undirected edges in unshielded triples following
 #' the orientation method
 #'
-#' @param state_order [a data frame]
-#' An optional d*(2-3) data frame giving the order of the ordinal categorical variables.
-#' It will be used during post-processing to compute the signs of the edges using partial
-#' linear correlation. 
-#' If specified, the data frame must have at least a "var_names" column, containing the
-#' names of each variable as specified by colnames(input_data). A "var_type" column may
-#' specify if each variable is to be considered as discrete (0) or continuous (1). And 
-#' the "levels_increasing_order" column contains a single character string with all of
-#' the unique levels of the ordinal variable in increasing order, delimited by a comma.
-#' If the variable is categorical but not ordinal, the "levels_increasing_order" column
-#' may instead contain NA.
+#' @param state_order [a data frame] An optional data frame providing extra
+#' information for variables. It must have d rows where d is the number of input
+#' variables, and the following structure (named columns):
+#'
+#' "var_names" (required) contains the name of each variable as specified
+#' by colnames(input_data).
+#'
+#' "var_type" (optional) contains a binary value that specifies if each
+#' variable is to be considered as discrete (0) or continuous (1).
+#'
+#' "levels_increasing_order" (optional) contains a single character string
+#' with all of the unique levels of the ordinal variable in increasing order,
+#' delimited by comma ','. It will be used during the post-processing to compute
+#' the sign of an edge using partial linear correlation. If the variable is
```

Comment:
It does not use partial linear correlation, but partial Spearman's rank correlation. Spearman's correlation assesses monotonic relationships (whether linear or not).

Reply:
These are some outdated docs I suppose, I rephrase only to make room for the new param so I didn't change anything regarding the content, maybe an overall revise of these docs is needed

Reply:
An issue should be opened to make sure we don't forget about this. I will create it.